src/stylesheets/main.css

```diff
@@ -249,8 +228,28 @@ p.important {
   background: var(--calitp-purple-4);
 }
 
-#reachout {
-  padding-top: 80px;
+.nav-pills {

```

Comment:
Note: This CSS will have to be refactored when the Filtering styling happens (cos the colors are different). But until then, it will be like this.